### PR TITLE
Remove scroll overlay, add mobile author profiles

### LINKED ISSUE #493: number of followers looks like link on collection page
[Origin page](https://beta.webcomponents.org/collection/samuelli/best-emoji)

### LINKED ISSUE #496: Rename inline demo editor to inline demo creator

### LINKED ISSUE #497: Author profile is too big
[Origin page](https://beta.webcomponents.org/author/webduinoio)

### LINKED ISSUE #499: All links on the introduction page should use https
[Origin page](https://beta.webcomponents.org/introduction)

### PR DESCRIPTION
**Pull request by samuelli**
Fixes #499 
Fixes #496 
Fixes #497 
Fixes #493

### CODE REVIEW

File: client/src/catalog-static.html [hunk 1]
```diff
@@ -94,10 +94,10 @@ <h1 class="page-title">Introduction</h1>
         <p>Many libraries already exist that make it easier to build web components. To dive in and create your own components, here are some you can try out:</p>
 
         <ul>
-          <li><a href="http://bosonic.github.io/">Bosonic</a> is a collection of components designed to meet the everyday needs of web developers.</li>
+          <li><a href="https://bosonic.github.io/">Bosonic</a> is a collection of components designed to meet the everyday needs of web developers.</li>
           <li><a href="https://www.polymer-project.org/1.0/">Polymer</a> provides a set of features for creating custom elements.</li>
-          <li><a href="http://skate.js.org/">SkateJS</a> is a JavaScript library for writing web components with a small footprint.</li>
-          <li><a href="http://x-tag.github.io/">X-Tag</a> is an open source JavaScript library that provides an interface for component development.</li>
+          <li><a href="https://skate.js.org/">SkateJS</a> is a JavaScript library for writing web components with a small footprint.</li>
```
Comment (abdonrd) [inline client/src/catalog-static.html]: Doesn't work. :/

File: client/src/catalog-static.html [hunk 2]
```diff
@@ -94,10 +94,10 @@ <h1 class="page-title">Introduction</h1>
         <p>Many libraries already exist that make it easier to build web components. To dive in and create your own components, here are some you can try out:</p>
 
         <ul>
-          <li><a href="http://bosonic.github.io/">Bosonic</a> is a collection of components designed to meet the everyday needs of web developers.</li>
+          <li><a href="https://bosonic.github.io/">Bosonic</a> is a collection of components designed to meet the everyday needs of web developers.</li>
           <li><a href="https://www.polymer-project.org/1.0/">Polymer</a> provides a set of features for creating custom elements.</li>
```
Comment (abdonrd) [inline client/src/catalog-static.html]: Maybe remove `/1.0/`?

File: client/src/catalog-static.html (hunk 1 above)
Comment (Nevraeka) [inline client/src/catalog-static.html]: cc @samuelli

File: client/src/catalog-static.html (hunk 2 above)
Comment (samuelli) [inline client/src/catalog-static.html]: Done

File: client/src/catalog-static.html (hunk 1 above)
Comment (samuelli) [inline client/src/catalog-static.html]: Fixed. Had to revert to http

**Comment by samuelli** [comment]
@Nevraeka: Yep, all the resources on our origin are enforced to be secure.

App-engine has a `secure: always` flag.
https://github.com/webcomponents/beta/blob/master/client/client.yaml

**Comment by Nevraeka** [comment]
@samuelli - @treshugart responded that the `.js.org` is the issue so they will not be updating that :(

**Comment by samuelli** [comment]
@Nevraeka @treshugart: I'll change it to point to GitHub since that's where the current webcomponents.org points to.

**Comment by treshugart** [comment]
Pretty sure it is, anyway. I'll have a look tomorrow and see what's up.

**Pull request merged**